techlibs/xilinx/synth_xilinx.cc

```diff
@@ -212,10 +208,6 @@ struct SynthXilinxPass : public ScriptPass
 				ise = true;
 				continue;
 			}
-			if (args[argidx] == "-iopad") {
```

Comment:
Perhaps leave this option in (as a noop) — it could already be in someone's synthesis script.

Reply:
adressed in 1937091f622a37d8050e5cc1e7c486707fd90b2f